Makefile

```diff
@@ -1,6 +1,7 @@
 GENERATED_DIR=$(shell pwd)/generated_python
-FINAL_DIR=gcloud/bigtable/_generated
+FINAL_DIR=$(shell pwd)/gcloud/bigtable/_generated
 GRPC_PLUGIN=grpc_python_plugin
+PROTOC_CMD=protoc
```

Comment:
I did this because I wanted to patch it locally and use the one from Linuxbrew/Homebrew, which for some reason is more updated than the one from the debian package repo (`apt-get install protobuf-compiler`)